run.sh

```diff
@@ -3,9 +3,10 @@
 access_id=${SUMO_ACCESS_ID:=$1}
 access_key=${SUMO_ACCESS_KEY:=$2}
 receiver_url=${SUMO_RECEIVER_URL:=https://collectors.sumologic.com}
-collector_name=${SUMO_COLLECTOR_NAME:=`hostname`}
+collector_name=${SUMO_COLLECTOR_NAME_PREFIX}${SUMO_COLLECTOR_NAME:=`hostname`}
```

Comment:
Can we assign a default value "collector_container-" for SUMO_COLLECTOR_NAME_PREFIX?